src/Components/Search/Previews/Grids/ArtistSearch/MarketingCollections.tsx

```diff
@@ -0,0 +1,101 @@
+import { data as sd } from "sharify"
+
+import { Box, color, Flex, Link, Sans, Serif, space } from "@artsy/palette"
+import { MarketingCollectionsPreview_marketingCollections } from "__generated__/MarketingCollectionsPreview_marketingCollections.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+
+interface MarketingCollectionsPreviewProps {
+  marketingCollections: MarketingCollectionsPreview_marketingCollections
+}
+
+const CollectionBox = styled(Box)<{ imageUrl: string }>`
+  margin-right: ${space(2)}px;
+  margin-bottom: ${space(2)}px;
+  width: 185px;
+  height: 80px;
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)),
+    url(${p => p.imageUrl}) center;
+
+  &:hover {
+    background: linear-gradient(
+        to bottom,
+        rgba(0, 0, 0, 0.5),
+        rgba(0, 0, 0, 0.5)
+      ),
+      url(${p => p.imageUrl}) center;
+  }
+`
+
+const CollectionTitle = styled(Serif).attrs({ display: "inline" })`
+  color: ${color("white100")};
+`
```

Comment:
#minor - it's slightly clearer to move styled-component definitions below the main react components. That way down the line when people view the code they can immediately get to business rather than having to wade through styles.